resources/dev_resources/EndpointResourcesList.json

```diff
@@ -2,7 +2,7 @@
     {
         "EndpointName":"Epic",
         "FileName":"EpicEndpointSources.json",
-        "URL":"https://open.epic.com/MyApps/EndpointsJson"
+        "URL":"https://open.epic.com/Endpoints/DSTU2"
```

Comment:
We also definitely want to process the R4 endpoints, so we should add those to the repo as well